docs/docsite/rst/community/steering/community_steering_committee.rst

```diff
@@ -161,10 +159,10 @@ When reviewing community collection `inclusion requests <https://github.com/ansi
 Community Working Group meetings
 ---------------------------------
 
-See the Community Working Group meeting `schedule <https://github.com/ansible/community/blob/main/meetings/README.md#wednesdays>`_. Meeting summaries are posted in the `Community Working Group Meeting Agenda <https://github.com/ansible/community/issues?q=is%3Aopen+label%3Ameeting_agenda+label%3Acommunity+>`_ issue.
+See the Community Working Group meeting `schedule <https://github.com/ansible-community/meetings/tree/main#wednesdays>`_. Meeting summaries are posted on the `Forum <https://forum.ansible.com/tags/c/workflow/meetbot/16/community-wg>`_.
 
 .. note::
 
-  Participation in the Community Working Group meetings is optional for Committee members. Decisions on community topics are made asynchronously in the `community-topics <https://github.com/ansible-community/community-topics/issues>`_ repository.
+  Participation in the Community Working Group meetings is optional for Committee members. Decisions on `community topics <https://forum.ansible.com/tags/c/project/7/community-wg>`_ are made asynchronously in the topics themselves.
 
-The meeting minutes can be found at the `fedora meetbot site <https://meetbot.fedoraproject.org/sresults/?group_id=ansible-community&type=channel>`_ and the same is posted to `Ansible Devel Mailing List <https://groups.google.com/g/ansible-devel>`_ after every meeting.
+The meeting minutes can be found at the `fedora meetbot site <https://meetbot.fedoraproject.org/sresults/?group_id=ansible-community&type=channel>`_ after every meeting.
```

Comment:
Ouch, we never posted them to ansible-devel... It's good to remove that... Also, the Fedora meetbot link is broken. How about simply removing this paragraph? A link to the meeting logs (their new place) can already be found at the start of this section.
```suggestion
```

Reply:
@felixfontein sounds good to me, committed, thanks, PTAL